modules/phase_field/include/materials/SwitchingFunctionMaterialMix.h

```diff
@@ -0,0 +1,40 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#ifndef SWITCHINGFUNCTIONMATERIALMIX_H
+#define SWITCHINGFUNCTIONMATERIALMIX_H
+
+#include "OrderParameterFunctionMaterial.h"
+
+// Forward Declarations
+class SwitchingFunctionMaterialMix;
+
+template <>
+InputParameters validParams<SwitchingFunctionMaterialMix>();
+
+/**
+ * Material class to provide the switching function \f$ h(\eta) \f$ for
+ * the KKS system.
+ *
+ * \see KKSPhaseChemicalPotential
+ * \see KKSCHBulk
+ */
+class SwitchingFunctionMaterialMix : public OrderParameterFunctionMaterial
```

Comment:
Please rename to `MixedSwitchingFunctionMaterial`; this should end on Material for consistency.